docs/_docs/analysis-types/known-vulnerabilities.md

```diff
@@ -49,4 +49,4 @@ VulnDB is a source of vulnerability intelligence that provides its own content.
 
 Dependency-Track contains an internal limiter which prevents repeated requests to remote services when performing
 vulnerability analysis. When a components Package URL or CPE is successfully used for a given analyzer, the action
-and the timestamp is recorded and compared to the interval throttle. The interval throttle defaults to 24 hours.
+and the timestamp is recorded and compared to the interval throttle. The interval throttle defaults to 12 hours.
```

Comment:
Can we rewrite to say analyzer results are cached? That would be more clear than "the action is being recorded" and "interval throttle".

_Dependency-Track contains an internal limiter which prevents repeated requests to remote services when performing
vulnerability analysis. When a components Package URL or CPE is successfully analyzed by a given analyzer, the result is cached. By default these cache entries expiry after 12h._

Reply:
Good point, I updated this part according to your suggestion. Thanks!